packages/pluggableWidgets/shrinking-header-web/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "shrinking-header-web",
+  "widgetName": "ShrinkingHeader",
+  "version": "1.0.0",
+  "description": "A fixed header that shrinks when scrolling down a page",
+  "copyright": "Mendix B.V.",
```

Comment:
I changed it a bit, but it's still property of Mendix, since I developed it during my crafting days. The only thing that differs is that it's community support.